src/components/bookForms.js

```diff
@@ -34,42 +35,33 @@ const Forms = () => {
       setCategory('');
     }
   };
-
-  const Selectbk = [{
-    value: '', label: 'category', disabled: true, selected: true,
-  },
-  { value: 'Action', label: 'Action' },
-  { value: 'Documentary', label: 'Documentary' },
-  { value: 'History', label: 'History' },
-  { value: 'Science', label: 'Science' },
-  { value: 'Medicine', label: 'Medicine' },
-  ];
   return (
-    <form onSubmit={eventSubmit} className="inputfield">
-
-      <input type="text" placeholder="Book Title" required id="title" name="title" value={title} onChange={eventTitle} />
+    <div className="addSection">
+      <hr />
+      <div className="addHeader">Add New Book</div>
+      <div className="input-cont">
+        <form onSubmit={eventSubmit} className="form-cont">
+          <input type="text" className="input-field" placeholder="Book Title" required id="title" name="title" value={title} onChange={onChange} />
 
-      <input type="text" placeholder="Book Author" required id="author" name="author" value={author} onChange={eventAuthor} />
+          <input type="text" className="input-field" placeholder="Book Author" required id="author" name="author" value={author} onChange={onChange} />
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: It would be nice if you could stretch the input fields to align with the cards from left to right so your design matches the original design. Kindly modify your styles :+1: but still it's not aligned properly please stick to the Zaplin design as much as possible. check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-05-28 04-13-42](https://github.com/emmanuelsteven/BookStore-React-Redux/assets/45226848/6b1016b7-9f8e-447a-af01-a75bb4d3d3a7) 

**Zaplin Design**
![Screenshot from 2023-05-28 04-13-14](https://github.com/emmanuelsteven/BookStore-React-Redux/assets/45226848/3ed4675f-2432-4a22-8f9c-f5431e3984a7)